src/services/ConfigService/ConfigService.go

```diff
@@ -3,6 +3,7 @@ package ConfigService
 import (
 	"log"
 	"io/ioutil"
+	log "github.com/sirupsen/logrus"
```

Comment:
If you rename `logrus` to `log` here, you have to remove the first imported `log`

Reply:
Yeah, forgot. to remove this, will do. Thanks